diff --git a/node_modules/kysely/dist/migration/file-migration-provider.js b/node_modules/kysely/dist/migration/file-migration-provider.js
index ed94aeb..8aa8051 100644
--- a/node_modules/kysely/dist/migration/file-migration-provider.js
+++ b/node_modules/kysely/dist/migration/file-migration-provider.js
@@ -1,5 +1,6 @@
 /// <reference types="./file-migration-provider.d.ts" />
 import { isFunction, isObject } from '../util/object-utils.js';
+import { pathToFileURL } from 'node:url';
 /**
  * Reads all migrations from a folder.
  *
@@ -32,7 +33,7 @@ export class FileMigrationProvider {
             const filePath = this.#props.path.join(this.#props.migrationFolder, fileName);
             const migration = this.#props.import
                 ? await this.#props.import(filePath)
-                : await import(/* webpackIgnore: true */ filePath);
+                : await import(/* webpackIgnore: true */ pathToFileURL(filePath).href);
             const migrationKey = fileName.substring(0, fileName.lastIndexOf('.'));
             // Handle esModuleInterop export's `default` prop...
             if (isMigration(migration?.default)) {
